### PR TITLE
[GraphQL] Allow unauthenticated queries

### PR DESCRIPTION
**Pull request by jamesdphillips**
## What is this change?

Permit requests that do not include an access token or API key,
this allows unauthenticated clients to run introspection queries or
query resources that do not require authorization, such as health,
version info, and sign-in methods.

**Notes**

* The existing API blocks unauthenticated users from accessing
resources guarded by applicable RBAC rules.
* For ease of consumers, an additional field has been added to query
results, indicating whether the query included authorization details.
* PR includes updates to the web application, that avoid any crashes
that this change might cause.

## How did you verify this change?

There is already a great deal of coverage for authorization so I did not
feel as if there was much room to extend, however I spent a great deal
of time manual testing.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,6 +27,15 @@ before checking if the asset has 1 or more builds.
 - Listing assets with no results returns an empty array.
 - Fixed a panic that could occur when creating resources in a namespace that
 does not exist.
+- [Web] Links to documentation now point to the version of the product being run
+instead of the latest; helpful when running an older version of Sensu.
+`sensu-backend`. These are used to take advantage of the embedded etcd's
```
Comment (ccressent) [inline CHANGELOG.md]: I think this line has been mangled.

File: CHANGELOG.md [hunk 2]
```diff
@@ -32,6 +32,13 @@ before checking if the asset has 1 or more builds.
 - Listing assets with no results returns an empty array.
 - Fixed a panic that could occur when creating resources in a namespace that
 does not exist.
+- [Web] Links to documentation now point to the version of the product being run
+instead of the latest; helpful when running an older version of Sensu.
+- Installing sensuctl commands via Bonsai will now check for correct labels
```
Comment (nikkixdev) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -32,6 +32,13 @@ before checking if the asset has 1 or more builds.
...
+before checking if the asset has 1 or more builds.
```
Comment (nikkixdev) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -32,6 +32,13 @@ before checking if the asset has 1 or more builds.
...
+- Listing assets with no results returns an empty array.
```
Comment (nikkixdev) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -32,6 +32,13 @@ before checking if the asset has 1 or more builds.
...
+- Fixed a panic that could occur when creating resources in a namespace that
```
Comment (nikkixdev) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -32,6 +32,13 @@ before checking if the asset has 1 or more builds.
...
+does not exist.
```
Comment (nikkixdev) [inline CHANGELOG.md]:
```suggestion
```

**Comment by jamesdphillips** [comment]
Thanks for merging @nikkixdev 🙏

**Pull request merged**